server/container/guice/guice-common/src/main/java/org/apache/james/modules/protocols/SMTPServerModule.java

```diff
@@ -42,6 +45,9 @@
 
     @Override
     protected void configure() {
+        bind(DNSServiceXBillImpl.class).in(Scopes.SINGLETON);
```

Comment:
I don't think it belong to SMTPServerModule, don't we have a DNSModule somewhere ?